website/src/components/docs/library/api/MethodSignature.tsx

```diff
@@ -0,0 +1,128 @@
+import styled from "@emotion/styled";
+import { useTypescriptSnippets } from "@site/src/components/TypeScriptToggle";
+import React, { FC } from "react";
+import { Parameter, Signature, Type } from "./getLibraryMethods";
+
+interface Props {
+  methodName: string;
+  signature: Signature;
+}
+
+const MethodSignature: FC<Props> = ({ methodName, signature }) => {
+  const showTypeScriptSnippets = useTypescriptSnippets();
```

Comment:
nit: inconsistent camel casing in `TypeScript` variable vs `Typescript` hook name.